src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started work!");
+    }
+    public void stopWork() {
```

Comment:
The `stopWork` method should have an `@Override` annotation to indicate that it overrides a method from the superclass `Machine`.